lgsm/functions/info_messages.sh

```diff
@@ -650,26 +650,44 @@ fn_info_message_ports(){
 
 	startparameterslocation="${red}UNKNOWN${default}"
 	# engines/games that require editing in the config file.
-	local ports_edit_array=( "avalanche2.0" "avalanche3.0" "Ballistic Overkill" "Barotrauma" "dontstarve" "Eco" "idtech2" "idtech3" "idtech3_ql" "lwjgl2" "Minecraft Bedrock" "Project Cars" "projectzomboid" "quake" "refractor" "realvirtuality" "renderware" "Stationeers" "teeworlds" "terraria" "unreal" "unreal2" "unreal3" "TeamSpeak 3" "Mumble" "7 Days To Die" "Vintage Story" "wurm")
+	local ports_edit_array=( "arma3" "ac" "bo" "bt" "dst" "eco" "idtech2" "idtech3" "idtech3_ql" "lwjgl2" "jc2" "jc3" "mcb" "mumble" "pc" "pz" "qw" "refractor" "renderware" "rw" "sb" "sdtd" "st" "ts3" "tw" "terraria" "unreal" "unreal2" "unreal3" "vints" "wurm")
 	for port_edit in "${ports_edit_array[@]}"; do
 		if [ "${shortname}" == "ut3" ]; then
 			startparameterslocation="${servercfgdir}/UTWeb.ini"
 		elif [ "${shortname}" == "kf2" ]; then
 			startparameterslocation="${servercfgdir}/LinuxServer-KFEngine.ini\n${servercfgdir}/KFWeb.ini"
-		elif [ "${engine}" == "${port_edit}" ]||[ "${gamename}" == "${port_edit}" ]; then
+		elif [ "${engine}" == "${port_edit}" ]||[ "${gamename}" == "${port_edit}" ]||[ "${shortname}" == "${port_edit}" ]; then
 			startparameterslocation="${servercfgfullpath}"
 		fi
 	done
 	# engines/games that require editing the start parameters.
-	local ports_edit_array=( "Avorion" "col" "goldsrc" "Factorio" "Hurtworld" "iw3.0" "ioquake3" "qfusion" "Rust" "scpsl" "scpslsm" "Soldat" "spark" "source" "starbound" "unreal4" "realvirtuality" "Unturned" "vh" )
+	local ports_edit_array=( "av" "col" "goldsrc" "fctr" "hw" "iw3.0" "ioquake3" "qfusion" "rust" "scpsl" "scpslsm" "sol" "spark" "source" "unreal4" "arma3" "unt" "vh" )
 	for port_edit in "${ports_edit_array[@]}"; do
 		if [ "${engine}" == "${port_edit}" ]||[ "${gamename}" == "${port_edit}" ]||[ "${shortname}" == "${port_edit}" ]; then
 			startparameterslocation="${configdirserver}"
 		fi
 	done
 	echo -e "${startparameterslocation}"
 	echo -e ""
+}
+
+fn_info_message_ports(){
 	echo -e "${lightblue}Useful port diagnostic command:${default}"
+	if [ "${shortname}" == "av" ]; then
+		echo -e "ss -tuplwn | grep AvorionServer"
+	elif [ "${engine}" == "source" ]; then
+		echo -e "ss -tuplwn | grep srcds_linux"
+	elif [ "${engine}" == "goldsrc" ]; then
+		echo -e "ss -tuplwn | grep hlds_linux"
+	elif [ "${shortname}" == "rw" ]; then
```

Comment:
we need here more cases for other java servers like minecraft/papermc/waterfall maybe we can use here the preexecutable is definded. Simular condition applys to games that need mono to start like vintage story.

for this case i would probably do something like this:
`echo -e "ss -tuplwn | grep (echo -e "${preexecutable}" | awk '{print $1}')"`

Extra special case is as well BF1042

Reply:
added java servers and bf1942